src/zoid/card-fields/component.jsx

```diff
@@ -197,6 +201,18 @@ export const getCardFieldsComponent : () => CardFieldsComponent = memoize(() : C
                     }
                 },
 
+                minLength: {
```

Comment:
I worry that `minLength` and `maxLength` are too generic. Could we rename them so its clear that they are used for the postalCode?
```suggestion
                postalCodeMinLength: {
```

Reply:
These map directly to the `<input>` element's `minlength` and `maxlength` attributes. So I'd prefer to keep them generic. This is an example of how they are set:
```js
cardFields.PostalCodeField({
    minLength: 5,
    maxLength: 8
});
``` 
So it should be pretty clear that these are for the `PostalCodeField`. Also, in the future we may want to allow merchant's to set them on other card fields. For example:
```js
cardFields.NameField({
    maxLength: 500
});
```
So it would be nice to keep them consistent.

Reply:
ahh I think I understand now. The card field props are generic in checkout-components and the implementation for each child component is in SPB.